cmd/native-btc/relayer_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gonative-cc/relayer/bitcoin"
+	"github.com/gonative-cc/relayer/dal"
+	"gotest.tools/assert"
+)
+
+func TestRelayerStart(t *testing.T) {
+	db := initTestDB(t)
+
+	config := Config{
+		DatabasePath: ":memory:",
+		BitcoinNode:  "mock-node",
+	}
+	relayer, err := NewRelayer(config, db)
+	assert.NilError(t, err)
+	relayer.btcClient = &bitcoin.MockClient{}
+
+	transactions := []dal.Tx{
+		{BtcTxID: 1, RawTx: "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0100f2052a010000001976a914000000000000000000000000000000000000000088ac00000000", Status: dal.StatusPending},
+		{BtcTxID: 2, RawTx: "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0200f2052a010000001976a914000000000000000000000000000000000000000088ac00000000", Status: dal.StatusPending},
+	}
+	for _, tx := range transactions {
+		err = db.InsertTx(tx)
+		if err != nil {
+			assert.NilError(t, err)
+		}
+	}
+
+	// Start the relayer in a separate goroutine
+	go relayer.Start()
```

Comment:
will need to check if relayer.Start will return error:
```go
go func() { 
  err := relayer.Start() 
  assert.NilError(t, err)
}()
```

Reply:
done